### PR TITLE
Vendor dependencies using glide.

### PR DESCRIPTION
**Pull request by alokmenghrajani**
- [ ] @csstaub 
- [ ] @mcpherrinm

### CODE REVIEW

**Comment by mcpherrinm** [comment]
Looks good, once we get Travis issues sorted.

We'll have to drop go 1.4 support, at least.

**Comment by csstaub** [comment]
Need to exclude `vendor/` in .gitignore. Will get created automatically by glide on build. Having it be in the repo causes problems.

File: README.md [hunk 1]
```diff
@@ -29,11 +29,12 @@ KeywhizFs will display all secrets under the top level directory of the mountpoi
 
 # Building
 
-Run `go build keywhizfs/main.go`.
+Install glide (`go get github.com/Masterminds/glide` or `brew install glide`)
+Run `make depends`.
```
Comment (csstaub) [inline README.md]: Don't you mean make build here?

**Comment by csstaub** [comment]
Need to add `github.com/stretchr/testify/assert` as dependency (used for testing)

File: README.md (hunk 1 above)
Comment (alokmenghrajani) [inline README.md]: yes, fixed.

**Comment by mcpherrinm** [comment]
:tada:

**Pull request merged**